PowerUp/app/src/main/res/layout/avatar_room.xml

```diff
@@ -37,81 +39,93 @@
         android:layout_width="@dimen/avatar_room_width"
         android:layout_height="@dimen/avatar_room_height"
         android:layout_alignTop="@id/face"
-        android:layout_toLeftOf="@id/hair"
+        android:layout_alignLeft="@id/eyes"
         android:src="@drawable/cloth1" />
 
     <ImageButton
-        android:id="@+id/faceRight"
+        android:id="@+id/eyeLeft"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignRight="@id/face"
-        android:layout_below="@id/face"
-        android:layout_marginTop="@dimen/face_right_margin_top"
-        android:src="@drawable/ic_keyboard_arrow_right_white_18dp" />
+        android:src="@drawable/ic_keyboard_arrow_left_white_18dp"
+        android:layout_below="@+id/eyes"
```

Comment:
Use `@id` here as you are referencing another element. Use `@+id` only when defining a new id